wallet/txauthor/author.go

```diff
@@ -95,7 +98,7 @@ func NewUnsignedTransaction(outputs []*wire.TxOut, relayFeePerKb dcrutil.Amount,
 			scriptSizers = append(scriptSizers, txsizes.P2PKHScriptSize)
 		}
 
-		maxSignedSize := txsizes.EstimateSerializeSize(scriptSizers, outputs, true)
+		maxSignedSize := txsizes.EstimateSerializeSize(scriptSizers, outputs, fetchChange.ScriptSize())
```

Comment:
Since this will return the same thing each time, and it's already been called above, we can just call it once at the beginning of the func and save the value to a local var.